server/run.sh

```diff
@@ -12,15 +12,19 @@ if [ ! -f "config.py" ]; then
   cp config.example.py config.py
 fi
 
+# delete to avoid some module import problems
+if [ -f "../optima/optima.pyc" ]; then
+  rm ../optima/optima.pyc
```

Comment:
this was a problem due to a one-off change, so i'm not sure we need to enshrine it in code...although if we do, shouldn't we just remove all .pyc files?